src/big/big_decimal.cr

```diff
@@ -235,7 +235,7 @@ struct BigDecimal < Number
   # BigDecimal.new(1).div(BigDecimal.new(2))    # => BigDecimal(@value=5, @scale=2)
   # BigDecimal.new(1).div(BigDecimal.new(3), 5) # => BigDecimal(@value=33333, @scale=5)
   # ```
-  def div(other : BigDecimal, precision = DEFAULT_PRECISION) : BigDecimal
+  def div(other : BigDecimal, precision : Int = DEFAULT_PRECISION) : BigDecimal
```

Comment:
This comes from way before this PR, but I would have expected `precision` to be a `BigInteger` in `BigDecimal` 🤔

Reply:
FWIW, [bigdecimal-rs](https://github.com/akubera/bigdecimal-rs), which this implementation is based on, does the same